pycognaize/tests/test_document/test_pdf.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from pycognaize.document.pdf import Pdf
+from pycognaize.tests.resources import RESOURCE_FOLDER
+
+class TestPdf(unittest.TestCase):
+    def setUp(self):
+        self.path1 = RESOURCE_FOLDER + '/snapshots/60f554497883ab0013d9d906'
+        self.src_id_1 = '6645954b30f92ce8b63b5dc1274ee829dcc322073d238955536ce497fc4149e3'
+
+        self.pdf_1 = Pdf(src_id=self.src_id_1,path=self.path1).pdf
+
+
+    def test_get_pdf(self):
```

Comment:
change unittets to add more coverage on our functiions and, do not test FITZ object please